src/Files.App/Utils/Storage/Operations/FilesystemHelpers.cs

```diff
@@ -255,18 +255,14 @@ public async Task<ReturnResult> RestoreItemsFromTrashAsync(IEnumerable<IStorageI
 				else if (operation.HasFlag(DataPackageOperation.Link))
 				{
 					// Open with piggybacks off of the link operation, since there isn't one for it
-					if (isTargetExecutable)
+					if (isTargetExecutable || isTargetPythonFile)
 					{
 						var items = await GetDraggedStorageItems(packageView);
+						if (isTargetPythonFile && SoftwareHelpers.IsPythonInstalled())
```

Comment:
```suggestion
						if (isTargetPythonFile && SoftwareHelpers.IsPythonInstalled())
```
Shouldn't this be `!SoftwareHelpers.IsPythonInstalled`?

Reply:
Yes, fixing now